part-configuration.tex

```diff
@@ -187,20 +187,245 @@ \section{Matching your business processes}
+
+@@@ Labor/Overhead has an inventory and expense account but no income account. Chris T
+
+\subsubsection{Non-tracking items}
+\label{subsubsec-coa-non-tracking-items}
+
+The items on this line relate to untracked (non stocked) items, i.e. services.
+
+\begin{description}[style=nextline]
+    \item[Income (IC\_income)] Adds the account to the income account selection list in
+    the service definition screen.
+    \item[Expense (IC\_expense)] Adds the account to the expense account selection list in
+    the service definition screen.
+    \item[Tax (IC\_taxservice)] Adds a check mark to the service definition screen for the
+    applicable account. See \charef{cha-taxes} for more details on how taxes work in LedgerSMB.
+\end{description}
+
+\subsubsection{Fixed assets}
+\label{subsubsec-coa-fixed-assets}
+
+\begin{description}[style=nextline]
+    \item[Fixed asset (Fixed\_Asset)] Marks the account as holding the original asset value for the fixed
+    assets module, for some classes of fixed assets.
+    \item[Depreciation (Asset\_Dep)] Marks the account as holding the cumulative depreciation amount
+    for the fixed assets module, for some classes of fixed assets.
+    \item[Expense (asset\_expense)] Adds the expense account to the selection list of the fixed assets
+    accounting module. See \secref{sec-business-processes-accounting-fixed-asset-accounting} for more details.
+    \item[Gain (asset\_gain)] Account to hold book value gain upon disposal of a fixed asset.
+    \item[Loss (asset\_loss)] Account to hold book value loss upon disposal of a fixed asset.
+\end{description}
 
 \subsection{Special accounts}
 \label{subsec-company-config-coa-special-accounts}
 
+Special accounts are set by going to \menupath{System \ma Defaults} and scrolling down to "Default Accounts".
+
+As you can see, in \figref{fig:system-default-accounts} the heading "3999 -- NET MARGIN" has been selected and is the heading which is used as the highest level for the P\&L and lowest level at which income and expenses are being reported in the balance sheet.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\graphicswidth]{images/system-default-accounts.png}
+    \caption{System Default Accounts}
+    \label{fig:system-default-accounts}
+\end{figure}
+
+The other special accounts are used as follows:
+
 \begin{itemize}
-\item AR/AP summary accounts
-\item 5 other special purpose accounts, see ``Defaults'' screen discussion
-\item sales tax accounts
+    \item Current earnings - Used to summarize the P\&L result as a single line on the Balance Sheet,
+    \item Inventory - Used to summarize the inventory value on the balance sheet.
+    \item Income - Used to summarize income on the Income Statement.
+    \item Cost of Goods Sold - Used to summarize the cost of goods on the Income Statement.
+    \item Foreign Exchange Gain - @@@ TODO
+    \item Foreign Exchange Loss - @@@ TODO

```

Comment:
These two accounts are used to account for value differences in foreign currencies when recording payments to invoices in foreign currencies. In case an invoice is issued at 1 EUR == 1 USD, but payments arrive when 1 EUR == 0.80 USD, the amount of USD paid is still the invoiced amount, but the EUR amount differs. The EUR difference ends up in the FX Gain and FX Loss accounts.

Reply:
Added.